### ISSUE TITLE
show alerts only when stdout is connected to terminal

### ISSUE THREAD

**Issue by pdawyndt** [unipept/unipept#317]
`@tvwillem`: have you been able to implement the server-side alerts in such a way that they are only shown by the command line tools if stdout is connected to the terminal? In the example below, printing the alert is fine. However, I would expect it does not show up in my files if I redirect the output to a file or through a pipe. Can the alert be postponed until the first terminal session output?

```
$ echo ENFVYIAK | unipept pept2prot -e
Unipept 0.4.0 is released!
sequence,uniprot_id,taxon_id
ENFVYIAK,P42654,3906
ENFVYIAK,Q96453,3847
ENFVYIAK,C6TH93,3847
ENFVYIAK,C6TM63,3847
ENFVYIAK,M5WGY1,3760
ENFVYIAK,I1LUM3,3847
ENFVYIAK,F6H2P0,29760
ENFVYIAK,E1U3Z1,3827
ENFVYIAK,I3SLC5,3880
ENFVYIAK,M0TAI1,214687
ENFVYIAK,I1M3M0,3847
ENFVYIAK,G7LIR4,3880
ENFVYIAK,M0TY03,214687
```

_[Original issue](https://github.ugent.be/unipept/unipept/issues/317) by @pdawyndt on Tue May 13 2014 at 20:39._
_Closed by an unknown user on Tue May 13 2014 at 20:49._

**Comment by bmesuere** [comment]
@pdawyndt I did not think of this, but it should be easily doable to make it behave like that. 

(there is also the -q flag to supress these messages)


_[Original comment](https://github.ugent.be/unipept/unipept/issues/317#issuecomment-9632) by ghost on Tue May 13 2014 at 20:41._

**Comment by bmesuere** [comment]
Fixed in 0.4.1


_[Original comment](https://github.ugent.be/unipept/unipept/issues/317#issuecomment-9635) by ghost on Tue May 13 2014 at 20:49._
